src/main/java/com/powsybl/openloadflow/ac/solver/NewtonRaphsonParameters.java

```diff
@@ -14,6 +14,13 @@
  */
 public class NewtonRaphsonParameters extends AbstractNewtonParameters<NewtonRaphsonParameters> {
 
+    public enum UnrealisticVoltageCheckBehavior {
```

Comment:
@murgeyseb can you add the documentation of the new parameter you have introduced. Or I can do it if you prefer. Just tell me !